middlewares/logger_test.go

```diff
@@ -20,7 +20,7 @@ type logtestResponseWriter struct{}
 
 var (
 	logger                  *Logger
-	logfileName             = "traefikTestLogger.log"
+	logfileName             = "/traefik/logger/test.log"
```

Comment:
Could you please also remove the global variable `logfilePath`? It suffices to live on the test function stack.